src/ovirtapi.js

```diff
@@ -339,6 +339,8 @@ OvirtApi = {
     return {
       id: snapshot.id,
       description: snapshot.description,
+      type: snapshot.snapshot_type,
+      date: snapshot.date,
       persistMemoryState: snapshot.persist_memorystate === 'true',
```

Comment:
I'd add
````javascript
  isActive: snapshot.snapshot_type === 'active',
````
here since the test will always be done at least once in sorting and probably again when rendering the snapshot's component item.